src/public/main.js

```diff
@@ -0,0 +1,35 @@
+'use strict';
+
+let registrationData;
+let occupations;
+let stateOptions;
+let occupationList = document.getElementById('occupationList');
+let statesList = document.getElementById('statesList');
+
+const populateOccupations = () => {
+  for (let i = 0; i < occupations.length; i++) {
+    occupationList.options[i] = new Option(`${occupations[i]}`);
+  }
+};
+
+const populateStates = () => {
+  for (let i = 0; i < stateOptions.length; i++) {
+    statesList.options[i] = new Option(`${stateOptions[i].name}`);
+  }
+};
+
```

Comment:
This function operates in the same manner, dynamically inserting **states** options into **statesList**.